app/config/collections.php

```diff
@@ -1190,6 +1190,17 @@
                 'array' => false,
                 'filters' => ['encrypt'],
             ],
+            [
+                '$id' => 'providerIdToken',
```

Comment:
What is Token ID? this concept is not required for the other +20 adapters we have, it would be great to keep consistency in order to reduce complexity and ease maintenance.